addons/l10n_in/models/account_invoice.py

```diff
@@ -98,3 +105,24 @@ def _l10n_in_get_warehouse_address(self):
         # TO OVERRIDE
         self.ensure_one()
         return False
+
+    def unlink(self):
+        db_name = self._cr.dbname
+        logging_vals = []
+        for move in self:
+            if move.country_code == 'IN':
+                history = Markup("<br/><br/>").join(Markup("<b>%s</b> on <small>%s</small> <br/> %s") % (m.author_id.display_name, fields.Datetime.to_string(m.date), m.l10n_in_audit_log_preview) for m in self.message_ids if m.message_type == 'notification')
+                history += Markup("<br/><br/> <small>Date time Format %s</small>") % (DATETIME_FORMAT)
```

Comment:
shouldn't this be translated? (if so, you can move the text outside of the markup call)

Reply:
okay, move outside for translation in the next commit

Reply:
this is report is shown to non-technically person to keep a history of deleted records